scripts/scripts.js

```diff
@@ -232,7 +232,9 @@ async function loadLazy(doc) {
   const element = hash ? doc.getElementById(hash.substring(1)) : false;
   if (hash && element) element.scrollIntoView();
 
-  loadHeader(doc.querySelector('header'), 'adp-header');
```

Comment:
@nithinasokan Would the scripts.js file be considered part of the immutable files? If so, can I raise a PR to include this change in the main ADP repo? It's a check for a metadata property "hide-header" that lets us create pages without the ADP header included

Reply:
Yes, scripts are immutable. Please submit a PR to adp repo and I'll approve it right away. Thanks.

Reply:
https://github.com/adobe/assets-distribution-portal/pull/242 @nithinasokan

Reply:
Merged and synced to GMO